nixos/modules/services/cluster/kubernetes/dashboard.nix

```diff
@@ -113,8 +134,8 @@ in {
         };
         spec = {
           ports = [{
-            port = 80;
-            targetPort = 9090;
+            port = 443;
```

Comment:
These port changes look like changes to user-facing behaviour, is that right? If so, we need a release notes for it at the very least.

Reply:
Yes, the ports changed upstream, so I chose to update them  in our manifest